### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by bmc-msft**

### CODE REVIEW

**Review by chkeita — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.0.0
+### Added
+* Agent: Added clarifying context to file system errors.  [#421](https://github.com/microsoft/onefuzz/pull/421)
+* CLI/Service: Significantly expanded the [events](docs/webhook_events.md) available for webhooks.  [#394](https://github.com/microsoft/onefuzz/pull/394)
+* Agent: Added `{setup_dir}` to [configuration values expansion](docs/command-replacements.md) [#417](https://github.com/microsoft/onefuzz/pull/417)
```
Comment (ranweiler) [inline CHANGELOG.md:11]:
```suggestion
* Agent: Added `{setup_dir}` to [configuration value expansion](docs/command-replacements.md) [#417](https://github.com/microsoft/onefuzz/pull/417)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
...
+* Agent: Added `{tools_dir}` [configuration values expansion](docs/command-replacements.md) to `{supervisor_options}` and `{supervisor_env}` [#444](https://github.com/microsoft/onefuzz/pull/444)
```
Comment (ranweiler) [inline CHANGELOG.md:12]:
```suggestion
* Agent: Added `{tools_dir}` [configuration value expansion](docs/command-replacements.md) to `{supervisor_options}` and `{supervisor_env}` [#444](https://github.com/microsoft/onefuzz/pull/444)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
...
+### Changed
+* CLI/Service: Migrated `onefuzz status top` to use [Webhook Events](docs/webhook_events.md).  (BREAKING CHANGE) [#394](https://github.com/microsoft/onefuzz/pull/394)
+* CLI/Service: New notification secrets, such as ADO tokens, are managed in Azure KeyVault and are no longer accessible to the user once created.  (BREAKING CHANGE) [#326](https://github.com/microsoft/onefuzz/pull/326), [#389](https://github.com/microsoft/onefuzz/pull/389)
+* CLI/Service: Updated multiple Python dependencies. [#426](https://github.com/microsoft/onefuzz/pull/426), [#427](https://github.com/microsoft/onefuzz/pull/427),  [#430](https://github.com/microsoft/onefuzz/pull/)
+
+### Fixed
+* Agent: Fixed triggering condition for new unique report events [#422](https://github.com/microsoft/onefuzz/pull/422)
+* Deployment: Mitigate issues related to deployments within conditional access policy scenarios. [#447](https://github.com/microsoft/onefuzz/pull/447)
+* Agent: Fixed an issue where unused nodes would stop requesting new work. [#459](https://github.com/microsoft/onefuzz/pull/459)
+* Service: Fixed dead-node cleanup. [#458](https://github.com/microsoft/onefuzz/pull/458)
```
Comment (ranweiler) [inline CHANGELOG.md:23]:
```suggestion
* Service: Fixed dead node cleanup. [#458](https://github.com/microsoft/onefuzz/pull/458)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.0.0
+### Added
+* Agent: Added clarifying context to file system errors.  [#421](https://github.com/microsoft/onefuzz/pull/421)
```
Comment (ranweiler) [inline CHANGELOG.md:9]: I think this should be PR 423.

Comment (ranweiler) [reply CHANGELOG.md:9]:
```suggestion
* Agent: Added clarifying context to file system errors.  [#423](https://github.com/microsoft/onefuzz/pull/423)
```

**Review by ranweiler — APPROVED** [review]

**Pull request merged**